src/components/SmallCapsBadge/index.tsx

```diff
@@ -0,0 +1,6 @@
+import styled from "@emotion/styled";
+import Badge from "@leafygreen-ui/badge";
```

Comment:
I think if we want all our badges follow this pattern it might be ok to just name this Badge and just reexport the leafygreen badge from here. 
```
import styled from "@emotion/styled";
import Badge, {Variant} from "@leafygreen-ui/badge";

 const SmallCapsBadge = styled(Badge)`
  text-transform: lowercase;
`;
export default SmallCapsBadge;
export {Variant} 
```

Reply:
This way we can just import like this.

```
import Badge, {Variant} from "components/Badge"
```
instead of 
```
import {Variant} from "@leafygreen-ui/badge";
import {Badge} from "components/SmallCapsBadge";
```